src/common/notification/NotificationComponent.module.css

```diff
@@ -0,0 +1,13 @@
+.notification {
+    width: 300px;
+    position: fixed;
+    top: 100px;
+    right: 0;
+}
+
+.notification p {
```

Comment:
Is this a sort of hack to style a `HDS` defined element?

Reply:
For some reason hds Notification component doesn't have className prop. So only way to restrict it's size is to use some sort of wrapper. I'm not sure if that counts as a hack.